melodic/distribution.yaml

```diff
@@ -1942,30 +1942,6 @@ repositories:
       url: https://github.com/ccny-ros-pkg/imu_tools.git
       version: melodic
     status: developed
-  industrial_core:
-    doc:
-      type: git
-      url: https://github.com/ros-industrial/industrial_core.git
-      version: melodic
-    release:
-      packages:
-      - industrial_core
-      - industrial_deprecated
-      - industrial_msgs
-      - industrial_robot_client
-      - industrial_robot_simulator
-      - industrial_trajectory_filters
-      - industrial_utils
-      - simple_message
-      tags:
-        release: release/melodic/{package}/{version}
-      url: https://github.com/ros-industrial-release/industrial_core-release.git
-      version: 0.7.0-0
-    source:
-      type: git
-      url: https://github.com/ros-industrial/industrial_core.git
-      version: melodic
```

Comment:
If you do the rename you can just change the branch name here instead of reverting.

Reply:
Yes, I understand.

As I wrote above: the PR was more a discussion PR than a real attempt at getting the Melodic release removed.